docs/troubleshooting.asciidoc

```diff
@@ -28,3 +28,59 @@ include::copied-from-beats/getting-help.asciidoc[]
 == Debug
 
 include::copied-from-beats/debugging.asciidoc[]
+
+== Common problems
+=== 503: Queue is full
+APM Server has an internal queue which stores documents until they can be delivered to Elasticsearch.
+It is designed this way to alleviate problems that might occur if Elasticsearch is intermittently unavailable or if a large spike of data arrives at the APM Server at the same time.
+
+When the internal queue is full, APM Server will return a HTTP 503 status with the message "Queue is full".
+
+A full queue generally means APM Server that the amount of data being collected by agents and sent to APM Server is greater than the amount of data that the Elasticsearch cluster can ingest.
+If Elasticsearch is unavailable for a prolonged period or if a sudden spike of data arrives at the APM Server, the queue can also fill up.
+
+To solve the "503: Queue is full" problem, you have a few options:
+
+* <<reduce-data,Reduce the amount of data collected>>
+* <<increase-ingestion-rate,Increase the ingestion rate of Elasticsearch>>
+* <<increase-queue-size,Increase the size of the APM Server queue.>>
+
+
+=== 503: Request timed out waiting to be processed
+
+There is a limit to the number of requests that the APM Server will process at the same time.
+The APM Server will return a HTTP 503 status with the message "Request timed out waiting to be processed" when the limit is reached and the request from an agent is blocked from delivering the request.
```

Comment:
`and the request from an agent is blocked from delivering the request.` 
This is the case where you are not enqueuing on time right? I think the last part of the sentence is not clear or maybe just `from delivering the request` nest to be removed.